cmds/ocm/commands/ocmcmds/common/inputs/types/dockermulti/type.go

```diff
@@ -14,8 +14,8 @@ func init() {
 const usage = `
 This input type describes the composition of a multi-platform OCI image.
 The various variants are taken from the local docker daemon. They should be 
-built with the buildx command for cross platform docker builds.
-The denoted images, as well as the wrapping image index is packed as OCI
+built with the "buildx" command for cross platform docker builds (see https://ocm.software/docs/tutorials/best-practices/#building-multi-architecture-images) .
```

Comment:
Sorry for nitpicking

```suggestion
built with the "buildx" command for cross platform docker builds (see https://ocm.software/docs/tutorials/best-practices/#building-multi-architecture-images).
```

Reply:
Nevermind. I just saw that the error is from my suggestion and fixed it